README.md

```diff
@@ -22,12 +22,27 @@
 ### Ok, I get that this is a problem, how to find out that I'm affected?
   
 * Failures in the server log of http requests that are making updates 
+Looking at the web server logs to look failures related to http requests that were doing updates. Any 4xx or 5xx errors related to PUT, POST, DELETE requests indicate that probably such issue occurred in the application.
```

Comment:
>Looking at the web server logs to look failures related to http requests that were doing updates

How about

Monitoring web server logs for failures related to http requests modifying data